azuredevops_serviceendpoint_federated/main.tf

```diff
@@ -44,3 +44,20 @@ resource "azuredevops_serviceendpoint_azurerm" "azurerm" {
   azurerm_subscription_id   = var.subscription_id
   azurerm_subscription_name = var.subscription_name
 }
+
+#
+# Approval
+#
+
+resource "azuredevops_check_approval" "this" {
+  count = length(var.approver_ids) > 0 ? 1 : 0
```

Comment:
please use a variable for enable check approval for example `var.check_approval_enabled`